_articles/we-dedicate-5-of-our-gross-revenue-to-environmental-causes-and-open-source-software.md

```diff
@@ -0,0 +1,86 @@
+
+Plausible Analytics script weights less than 1 KB, which is 45 times smaller than the Google Analytics Global Site Tag script. Plausible will reduce your page weight, keep your site loading fast and reduce your electricity usage.
+
+A site with 10,000 monthly visitors can save more than 4.5 kg of CO2 emissions per year by replacing Google Analytics with Plausible Analytics. And with Plausible now installed on more than 10,000 sites and counting hundreds of millions of page views every month, this adds up. You can read more about what makes Plausible a [lightweight web analytics](https://plausible.io/lightweight-web-analytics) solution.
+
+The environmental causes and projects we will donate to remain to be decided. We're thinking of wildlife preservation, clean water and clean air initiatives. We'll take time later this year to pick out projects that make the most sense to us.
+
+### Open source software projects 
+
+On the open source side of things, we want to donate to the projects we stand on the shoulders of. The full list of projects remains to be decided.
+
+Running an open source project, we're very aware of how tough it is to [pay your rent](https://plausible.io/blog/open-source-funding) when the only accepted funding model is donations. Maintaining open source software is a thankless, time-consuming job. We released our code on GitHub and made it easy to self-host on principle, not because it's a profitable business.
+
+We had three $5/month donations for Plausible self-hosted in the first six months of accepting contributions and in the three months or so since that, I've been on a personal quest to try and improve that number.
+
+We're now up on nine $5/month donations and we're grateful to the people who decide to donate. Each donation helps contribute back to the Plausible community and the sustainability of our project.
+
+The fact is that it's easier to find a Google Analytics user, tell them about Plausible, let them have the trial and become a paying subscriber than it is to get a person who's self-hosting Plausible to [donate to the long-term sustainability of the project](https://github.com/sponsors/plausible). 
+
+From speaking to other open source maintainers, we are not an exception to this phenomenon. Possible reasons for why this may be are a topic for another post but it means that we're aware of this major issue that other open sources projects face too.
+
+If we want to see more open source software that can compete with the alternatives made by surveillance capitalism, users and projects that can afford to should donate and pay for open source software. And we are doing so too.
+
+## Plausible 5% fund: Numbers to date
+
+We will keep this page updated with the latest numbers and we'll also announce initiatives and projects we will donate to when we have finalized those. 
+
+"Plausible 5% for the environment and open source" donation fund to date:
+
+Month | Donation
```

Comment:
Hmm the table looks weird, I will take a look at styling tables

Reply:
thanks! cannot figure out how to center the markdown table :)